src/common/k8s-api/api-manager.ts

```diff
@@ -27,7 +27,11 @@ import type { KubeApi } from "./kube-api";
 import type { KubeObject } from "./kube-object";
 import { IKubeObjectRef, parseKubeApi, createKubeApiURL } from "./kube-api-parse";
 
-export class ApiManager {
+export interface IGettableStore {
+  getStore<TKubeObjectStore extends KubeObjectStore<KubeObject>>(api: string | KubeApi<KubeObject>): TKubeObjectStore | undefined;
```

Comment:
@jakolehm I recently stumbled across the following rule https://github.com/microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md

Would it be a good idea to slowly move towards that?

Reply:
Makes sense to me :)